packages/core/src/page.js

```diff
@@ -35,13 +35,18 @@ export class Page {
     this.log.debug('Page closed', this.meta);
   }
 
+  async reload() {
+    await this.session.send('Page.reload');
+    this.log.debug('Page reloaded', this.meta);
+  }
+
   // Resize the page to the specified width and height
-  async resize({ width, height }) {
-    this.log.debug(`Resize page to ${width}x${height}`);
+  async resize({ width, height, deviceScaleFactor = 1, mobile = false }) {
+    this.log.debug(`Resize page to ${width}x${height} at ${deviceScaleFactor}x scale`);
```

Comment:
What do you think about

```suggestion
    this.log.debug(`Resize page to ${width}x${height} @${deviceScaleFactor}x`);
```

So it would read `Resize page to 600x1080 @2x` for example

Reply:
I like it better than what I have